modules/monitoring-running-metrics-queries.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * monitoring/cluster-monitoring/examining-cluster-metrics.adoc
+
+[id="running-metrics-queries_{context}"]
+= Running metrics queries
+
+You begin working with metrics by entering one or several Prometheus Query Language (PromQL) queries.
```

Comment:
s/You begin working with metrics by/You can work with metrics by/

Reply:
My wording indicates that this is the first stage of a multi-stage process (working with metrics), which IMO is better. "You can work with metrics" refers to the whole process.